cmd/influx/main.go

```diff
@@ -242,3 +242,23 @@ func newLocalKVService() (*kv.Service, error) {
 
 	return kv.NewService(zap.NewNop(), store), nil
 }
+
+func getOrgID(orgSVC influxdb.OrganizationService, id string, name string) (influxdb.ID, error) {
+	if id != "" {
+		influxOrgID, err := influxdb.IDFromString(id)
+		if err != nil {
+			return 0, fmt.Errorf("invalid org ID provided: %s", err.Error())
+		}
+		return *influxOrgID, nil
+	} else if name != "" {
```

Comment:
nit: you don't need the else here, you can just do

```go
if id != "" {
.... yada yada
}
if name != "" {
.. yada yada
}
return yada yada
```

this is the usual approach in most go libs I've seen/authored 🤔

Reply:
oh okay. the former communicates to me that either one should exist or the other. nbd though, i can update it.